### PR TITLE
Add more info to signup rake task output

### PR DESCRIPTION
**Pull request by fimac**
Updates the activestash signup rake task to provide more info the user:

- where they are being redirected to
- next steps

Before

Screen Shot 2022-07-22 at 7 59 45 am

After

Screen Shot 2022-07-22 at 8 01 59 am

Notion card https://www.notion.so/cipherstash/ActiveStash-improve-rake-signup-task-78fe2a1ac319476c9be49a302f8af09c

### CODE REVIEW

**Review by mpalmer — APPROVED** [review]

File: lib/tasks/active_stash.rake [hunk 1]
```diff
@@ -27,7 +27,23 @@ namespace :active_stash do
...
     info("")
-    info("Opening your browser to complete signup.......")
+    info("")
+    info("")
+    info("You are being redirected to https://console.cipherstash.com/?signup=active-stash to complete your signup..")
+    info("")
+    info("")
+    info("NEXT STEPS:")
+    info("")
+    info("1. Signup with your github account or email.")
```
Comment (CDThomas) [inline lib/tasks/active_stash.rake:37]:
```suggestion
    info("1. Sign up with your GitHub account or email.")
```

Minor nits:
* I think it's "sign up" as a verb and "signup" as a noun
* Capitalisation for GitHub

File: lib/tasks/active_stash.rake [hunk 2]
```diff
@@ -27,7 +27,23 @@ namespace :active_stash do
...
+    info("")
+    info("")
+    info("2. Grab your workspace id from the signup confirmation page")
```
Comment (CDThomas) [inline lib/tasks/active_stash.rake:40]:
```suggestion
    info("2. Grab your workspace ID from the signup confirmation page")
```

File: lib/tasks/active_stash.rake [hunk 3]
```diff
@@ -27,7 +27,23 @@ namespace :active_stash do
...
+    info("")
+    info("")
+    info("3. Login using this rake command with your workspace id: rake active_stash:login[<your workspace id>]")
```
Comment (CDThomas) [inline lib/tasks/active_stash.rake:43]:
```suggestion
    info("3. Log in using this Rake command with your workspace ID: rake active_stash:login[<your workspace ID>]")
```
A couple more nits: 
* Same rule as above applied to "log in" vs "login"
* Capitalisation for ID

File: lib/tasks/active_stash.rake (hunk 1 above)
Comment (fimac) [reply lib/tasks/active_stash.rake:37]: Thanks @CDThomas , i'll do a quick follow up. Sorry for being so trigger happy 😆

**Pull request merged**